pkg/api/types.go

```diff
@@ -2049,10 +2049,11 @@ type SecurityContext struct {
 	// RunAsUser is the UID to run the entrypoint of the container process.
 	RunAsUser *int64 `json:"runAsUser,omitempty"`
 
-	// RunAsNonRoot indicates that the container should be run as a non-root user.  If the RunAsUser
-	// field is not explicitly set then the kubelet may check the image for a specified user or
-	// perform defaulting to specify a user.
-	RunAsNonRoot bool
+	// RunAsNonRoot indicates that the container should be run as a non-root user.
+	// Optional.  If true, the Kubelet will validate the image at runtime to ensure that it
+	// does not run as UID 0 (root) and fail to start the container if it does.
+	// If unset or false, no such validation will be performed.
```

Comment:
I'm thinking ahead to when PodSecurityContext can override an unspecified value.